makefile

```diff
@@ -160,7 +163,7 @@ $(BUILD_DIR)/%.o: %.s makefile | $(BUILD_DIR)
 	@echo ""
 
 $(BUILD_DIR)/$(TARGET).elf: $(OBJECTS) makefile
```

Comment:
Can you do like is in PIL, MC and BMS and build the submodule in the makefile. Just check over in PIL makefile probably. You add a target called `libs` where it goes into the wloop can folder and makes the library and then just make the .elf file require the libs target. Also make sure to add the wloop can make clean into the psyduck make clean as well